scripts/prepare.sh

```diff
@@ -5,7 +5,7 @@ type glide >/dev/null 2>&1 || (echo "installing Glide" && curl https://glide.sh/
 
 export GOPATH="$(pwd)/src/datadog-firehose-nozzle/:$GOPATH"
 
-DEFAULT_NOZZLE_VERSION=1.3.0
+DEFAULT_NOZZLE_VERSION=1.4.0
```

Comment:
Since there is a breaking change, should we make it `2.0.0` ?

Reply:
Hmm, I already released 1.4.0 on the nozzle repo. Do you think this is a big issue? IMO most people get the nozzle through the release published here or through the tile, so I don't think that version is hugely important.

Reply:
Well the release is just a tag, there would be no harm in updating it. Agreed though this one there is not too important (although people deploying the nozzle as an app might prefer to use the source code release directly).
I'm more concerned about the `v78` of the `nozzle-release` and the tile actually.

Reply:
That said I don't really have a precise idea of what we should do here 😕

Reply:
AFAIK the practice is that if you're using a single number, every release is considered to be a major release and potentially breaking, so I don't see this as a huge issue (and I will take care to document this carefully).

Reply:
Yeah I thought about that too, sounds good to me 👍